### PR TITLE
Fix for "Should not show the hint of the Clarity icon #734"

### PR DESCRIPTION
**Pull request by chandranive**
Changed the css tooltips.clarity.scss, to add "pointer-events: none;".

### CODE REVIEW

**Comment by chandranive** [comment]
Signed-off-by: Nivedita Chandra <chandran@vmware.com>

File: src/clarity-angular/tooltips/_tooltips.clarity.scss [hunk 1]
```diff
@@ -186,6 +186,7 @@ $clr-tooltip-adjusted-margin: $margin-value + $arrow-width * 2 !default;
     .tooltip {
         & > .clr-icon{
```
Comment (mathisscott) [inline src/clarity-angular/tooltips/_tooltips.clarity.scss]: `.clr-icon` isn't a class that is used with the Clarity Icon library. It's legacy from before our custom element based Icon framework.

Also, you probably want the pointer-events to be on the `svg` element inside the Clarity Icon.

So this line should be more like...

```
.tooltip {
  & > .clr-icon {
    margin-right: 0;
  }

  clr-icon > svg {
    pointer-events: none;
  }
}
```

**Comment by mathisscott** [comment]
`Signed-off-by: Nivedita Chandra <chandran@vmware.com>` needs to be the last line of your commit message. Rebase off master and add that to your commit message and then force push up to your forked branch.

File: src/clarity-angular/tooltips/_tooltips.clarity.scss (hunk 1 above)
Comment (chandranive) [inline src/clarity-angular/tooltips/_tooltips.clarity.scss]: Thanks @mathisscott for your suggestion. I have made the required changes and done a commit. Do I need to have a new Pull request too?

**Pull request closed without merging**